src/qt/guiutil_font.cpp

```diff
@@ -0,0 +1,616 @@
+// Copyright (c) 2014-2025 The Dash Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <qt/guiutil_font.h>
+
+#include <tinyformat.h>
+#include <util/system.h>
+
+#include <qt/guiutil.h>
+
+#include <QApplication>
+#include <QDebug>
```

Comment:
forgotten to remove from src/qt/guiutil.cpp:

```diff
-#include <QDebug>
-#include <QPointer>
```

Reply:
Resolved in latest push